executor/ddl_test.go

```diff
@@ -183,6 +183,15 @@ func (s *testSuite) TestAddNotNullColumnNoDefault(c *C) {
 	tk.MustExec("set sql_mode=''")
 	tk.MustExec("insert nn (c1) values (3)")
 	tk.MustQuery("select * from nn").Check(testkit.Rows("1 0", "2 0", "3 0"))
+
+	tk.MustExec("alter table nn add column (c3 int default -1, c4 int)")
```

Comment:
1. I mean the test like TestMySQLErrorCode, you need to check the error code in line196.
2. In `testAddMultipleColumns` only one case about position test, and it does not involve the index.
3. The new test of `TestAlterTableAddMultipleColumns` is OK.